_includes/community/com-sql-query-run.md

```diff
@@ -0,0 +1,9 @@
+| Method | Description | Further information |
+|---|---|---|
+| CLI | Run SQL statements from the FeatureBase CLI system | [Enable SQL CLI](/docs/community/com-config/com-config-sql-cli-enable) |
+| FeatureBase API | FeatureBase provides an API which can be used to execute SQL statements | [Community API](/docs/community/com-api/com-api-home) |
```

Comment:
this cannot execute SQL. Only PQL

Reply:
Now I know the truth... it's been updated :)